content/source/docs/enterprise/private/logging.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "enterprise2"
 page_title: "Terraform Enterprise Logging"
-sidebar_current: "docs-enterprise2-home"
+sidebar_current: "docs-enterprise2-private-installer"
```

Comment:
This should be `docs-enterprise2-logging`, right? (Man, I really want to get rid of this sidebar_current business.)